local_planner/CMakeLists.txt

```diff
@@ -227,12 +227,21 @@ if(CATKIN_ENABLE_TESTING)
 	                                      test/test_planner_functions.cpp
                                         test/test_star_planner.cpp
```

Comment:
The indentation seems funky in here, could you take the opportunity to replace all the tabs with 4 spaces?